### ISSUE TITLE
Install?

### ISSUE THREAD

**Issue by JohnBergman** [EndzoneSoftware/UmbracoSocialPluginAdapters#1]
I have been trying to get this to work... can you shed some light on exactly how to install this package?  It looks awesome!

**Comment by barryokane** [comment]
Hi John,

Have you seen http://endzonesoftware.com/installing-usplit/ and http://endzonesoftware.com/running-ab-experiments-like-pro-usplit/ ?

There is also a recent uHangout: https://www.youtube.com/watch?v=WQysVNLyQM8

Barry

PS: there is also this support: http://endzonesoftware.com/usplit-ab-testing-for-umbraco/

**Comment by JohnBergman** [comment]
I have, (BTW, like the usplit package 👍 )... I am looking for information about the socialplugin adapters.

**Comment by barryokane** [comment]
Ah, apologies, wrong plugin links!

Perhaps this will help? http://endzonesoftware.com/jquery-social-stream-plugin-with-dontnet/

**Comment by JohnBergman** [comment]
I'll take a look, do I need to get the code from designchemical and install it as well, or is this stand alone?

**Comment by barryokane** [comment]
Yes, you need the actual jquery plugin as well, this package simply adapts it from PHP to Umbraco (.Net)
